woocommerce/woocommerce-android
# Unified Login: Site Address Flow

This PR is part of an epic to implement the unified login flow. This particular flow implements the site address login flow and includes the following scenarios:

- Clicking "Enter store address" opens the site address screen
- Login with email and using magic link to finish the login process
- Login with email and password to finish the login process
- Login with store credentials - match to WPcom account and use magic link to verify email and complete the login process
- Login with store credentials - login wtih WPcom email and password to complete the login process

Not included:
- Since the login with site address doesn't use the store selector, that screen has not been updated to the new styles yet. 

## Screenshots

Prologue | site address screen | email screen 
-- | -- | -- 
![Screenshot_1598306824](https://user-images.githubusercontent.com/5810477/91107554-8f52ff80-e643-11ea-8872-df65b2cf13ad.png)|![Screenshot_1598311156](https://user-images.githubusercontent.com/5810477/91107584-a0037580-e643-11ea-88ae-71c0b9481740.png)|![Screenshot_1598313299](https://user-images.githubusercontent.com/5810477/91107622-b4e00900-e643-11ea-8191-17de80201612.png)|

 magic link request | magic link sent | password
-- | -- | --
![Screenshot_1598311173](https://user-images.githubusercontent.com/5810477/91107638-bc9fad80-e643-11ea-918c-9ab2735f357f.png)|![Screenshot_1598311179](https://user-images.githubusercontent.com/5810477/91107642-be697100-e643-11ea-9ede-a787981ceabe.png)|![Screenshot_1598311197](https://user-images.githubusercontent.com/5810477/91107669-d3460480-e643-11ea-97cd-3e9b92b13dbc.png)

store creds | verify with magic link | magic link sent
-- | -- | --
![Screenshot_1598311797](https://user-images.githubusercontent.com/5810477/91107747-025c7600-e644-11ea-85f3-d1489265d2ee.png)|![Screenshot_1598311828](https://user-images.githubusercontent.com/5810477/91107749-04263980-e644-11ea-9f0c-2a00adb4ab7b.png)|![Screenshot_1598311179](https://user-images.githubusercontent.com/5810477/91107754-0a1c1a80-e644-11ea-8848-caf728319276.png)

## To Test
- Verify entering site address and email, then logging in with magic link
- Verify entering site address, then entering store credentials and using magic link
- Verify entering site address, then entering store credentials and using password


Update release notes:
